docs/helpers/custom_helpers.rb

```diff
@@ -0,0 +1,48 @@
+require 'yaml'
+
+module CustomHelpers
+
+  def create_navigation
+  	navigation = ""
+  	topic_map = YAML.load_file('build/_topic_map.yml')
+  	topic_map["Topics"].each_with_index { |topic, index|
+  		if topic["Dir"].nil?
+  			next "<li><a class=\"\" href=\"/#{topic["File"]}.html\">&nbsp;#{topic["Name"]}</a></li>"
+  		end
+
+  		expanded = ""
+  		subnav = ""
+  		entries = ""
+  		unless topic["Topics"].nil?
+            topic["Topics"].each { |subtopic|
```

Comment:
could we use `do ... end` format here instead of `{...}`? less intuitive from Ruby point of view